packages/fabric8-ui/src/app/dashboard-widgets/space-settings-widget/space-settings-widget.component.less

```diff
@@ -0,0 +1,61 @@
+@import (reference) '../../../assets/stylesheets/shared/osio.less';
+
+.f8-small-card {
+  height: 200px;
+  &-body {
+    min-height: 130px;
+  }
+  .card-pf-footer.space-description-footer {
+    background: @color-pf-white;
+    padding: 14px 20px 10px;
+  }
+}
+
+.f8-space-settings {
+  &-data-links {
+    margin-top: 10px;
+    > li {
+      margin-top: 10px;
+    }
+  }
+}
+
+.edit-space-description {
+  &-widget {
+    display: flex;
+    align-items: center;
+    height: 130px;
+    color: @color-pf-black-200;
+    .space-description {
+      position: relative;
+      top: 10px;
+      height: 70px;
+      margin: 5px 0;
+      resize: none;
+      line-height: 1.5;
+    }
+  }
+  &-btn {
+    font-size: 12px !important;
+    margin-left: 5px;
+    text-align: center;
+    width: auto;
+    padding-right: 5px !important; /* stylelint-disable-line declaration-no-important */
+    padding-left: 5px !important; /* stylelint-disable-line declaration-no-important */
+    .pointer;
```

Comment:
I don't think we need these style properties to be defined here. These properties are already defined in Patternfly. I checked and buttons look fine without these properties.

Reply:
removed it

Reply:
@karthikjeeyar Style properties are still here